bgp/tests/proto/policyval/policy_validation.proto

```diff
@@ -0,0 +1,51 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package policy_validation;
```

Comment:
is reason with a proto and not just a struct?

Reply:
So there is another BGP policy testing framework that the user of this is looking at that currently uses this as the test interface. I'm simply using it here for uniformity and so I don't need to re-invent this. If there is no need for using a proto we can later change it, but it doesn't seem to have any harm.